src/js/other-websites/add_input_icon.js

```diff
@@ -54,35 +45,25 @@ function handleKeydownEvents(e) {
 
   if (e.key === "ArrowDown" || (e.key === "Tab" && e.shiftKey === false)) {
```

Comment:
I think this is intended to allow keyboard navigation, but as indicated in another comment already, that doesn't work at all for me atm.

Reply:
This function moved to the other file, `inpage_menu.js` in 4344dc8e29ad93376f4af2c58eebb7a81182ed08

Note: I did remove the tab watching from this function as it was not necessary.